frontend/hub/collections/UploadCollection.tsx

```diff
@@ -38,6 +61,75 @@ export function UploadCollectionByFile() {
   const repositories = useRepositories();
   const navigate = useNavigate();
   const onCancel = () => navigate(-1);
+  const toolbarFilters = useRepoFilters();
+  const tableColumns = useRepositoriesColumns();
+  const [onlyStaging, setOnlyStaging] = useState(true);
+  const [selectedRepo, setSelectedRepo] = useState<{ name: string } | null>(null);
```

Comment:
Are we interested in the name only or the whole repository data?

Reply:
For now, only name. But i think its better to pass { name : string }, so we will be able to easily add another data if needed in future.

Reply:
Future is now, I need to add pulp_href :D